src/test/java/org/jfrog/bamboo/builder/BuilderDependencyHelperTest.java

```diff
@@ -0,0 +1,26 @@
+package org.jfrog.bamboo.builder;
+
+import org.apache.commons.collections.map.HashedMap;
+import org.jfrog.bamboo.context.Maven3BuildContext;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+
+public class BuilderDependencyHelperTest {
+
+    private BuilderDependencyHelper builderDependencyHelper;
+
+    @Test
+    public void shouldNotThrowExceptionForEmptyTempDir() throws IOException {
+        builderDependencyHelper = new BuilderDependencyHelper("artifactoryBuilderTest");
+        builderDependencyHelper.downloadDependenciesAndGetPath( new File(""), "TEST-ART-BAMB",new Maven3BuildContext(new HashedMap()),"something" );
```

Comment:
We can use native Java module for that:
```suggestion
        builderDependencyHelper.downloadDependenciesAndGetPath(new File(""), "TEST-ART-BAMB", new Maven3BuildContext(new HashMap<>()), "something");
```